spec/fixtures/examples/ead_component.xml

```diff
@@ -1,25 +0,0 @@
-<c id="ref12" level="item">
-    <did>
-        <unittitle>[ACT UP Paris]</unittitle>
-        <container id="cid17653001" type="Box" label="Mixed materials">1</container>
-        <container parent="cid17653001" type="Folder">1</container>
-        <unitdate normal="1992/1992">August 19, 1992</unitdate>
-        <origination label="creator">
-          <persname source="local">Bytsura, Bill</persname>
-          <persname source="naf">Belfrage, Sally, 1936-</persname>
-          <corpname rules="Anglo-American_Cataloguing_Rules__2nd_ed." source="naf">Kings County (N.Y.). Board of Supervisors.</corpname>
-          <famname source="naf">Lefferts family</famname>
-        </origination>
-        <controlaccess>
-          <genreform source="aat">Clippings (information artifacts)</genreform>
-          <genreform source="aat">Correspondence.</genreform>
-          <geogname source="lcsh">Cambridge (Mass.)</geogname>
-          <geogname source="lcsh">Spain |x History |y Civil War, 1936-1939 |x Atrocities.</geogname>
```

Comment:
@barnabyalter @NYULibraries/finding-aids 

Barnaby, et al.,

Removing these component fixtures removes any testing we have of the subfield delimiter removal code. I was just adding to these to ensure that subjects were processed correctly, too

Shouldn't we go through existing specs & fixtures & make sure these new generic "kitchen sink" files cover all the cases that were covered by the previous tests?

If not, should I just not bother testing the subject-subfield code except in the subfield spec itself? 

Thanks,
-Corey